.github/actions/pr-review/pr_review.py

```
@@ -1532,8 +1585,222 @@ def _line_context(content: str, line: int | None) -> str:
+    {
+        "actual", "against", "already", "candidate", "change", "check", "code", "could",
+        "current", "does", "every", "finding", "from", "have", "into", "must", "only",
+        "description", "descriptions", "path", "release", "require", "review", "same", "should", "than", "that", "their",
+        "this", "value", "when", "where", "which", "with", "without",
+    }
+)
+
+
+def _evidence_terms(finding: Finding, _context: str) -> list[str]:
+    """Choose bounded, code-shaped terms that can locate consumers and tests."""
+    candidate_source = " ".join((finding.title, finding.why, finding.fix))
```

Ruff (0.16.1)

[warning] Consider `f"{finding.title} {finding.why} {finding.fix}"` instead of string join

Replace with `f"{finding.title} {finding.why} {finding.fix}"`

(FLY002)